src/mainboard/emulation/qemu-fsp/src/romstage.rs

```diff
@@ -0,0 +1,10 @@
+use crate::halt;
+use payloads::external::zimage::PAYLOAD;
+
+pub fn romstage() -> ! {
```

Comment:
Where is this invoked? I'm probably missing something foundational in how oreboot is put together.

Reply:
Good question. I also removed https://github.com/oreboot/oreboot/blob/master/src/mainboard/emulation/qemu-q35/src/romstage.rs as well.

It looks like the arm qemu mainboards are using their romstage functions. It'll be worth looking across these qemu targets sometime and refactoring some of the common bits.